test/Elastica/Base.php

```diff
@@ -123,7 +123,7 @@ protected function _createIndex($name = null, $delete = true, $shards = 1)
 
         $client = $this->_getClient();
         $index = $client->getIndex('elastica_'.$name);
-        $index->create(['index' => ['number_of_shards' => $shards, 'number_of_replicas' => 0]], $delete);
+        $index->create(['index' => ['number_of_shards' => $shards, 'number_of_replicas' => 2]], $delete);
```

Comment:
that's a good point don't know why I choose 2 :) I'm executing tests again on my local dev-machine than I will push it to remote